### ISSUE TITLE
project-installcheck can't handle download-on-demand repos

### ISSUE THREAD

**Issue by Vogtinator** [openSUSE/openSUSE-release-tools#2269]
`openSUSE:Factory/snapshot` -> `openSUSE:Tumbleweed/standard` are download-on-demand now, which the project-installcheck script does not like:

```
  nothing provides libc.so.6 needed by gmenudbusmenuproxy-5.17.80git.20191017T054225~e8041bb4b-ku.11.1.i586
  nothing provides libc.so.6(GLIBC_2.0) needed by gmenudbusmenuproxy-5.17.80git.20191017T054225~e8041bb4b-ku.11.1.i586
  nothing provides libc.so.6(GLIBC_2.1.3) needed by gmenudbusmenuproxy-5.17.80git.20191017T054225~e8041bb4b-ku.11.1.i586
...
```

**Comment by coolo** [comment]
@DimStar77 promised this change would make everything so much easier, so I'm awaiting a patch :)

**Comment by DimStar77** [comment]
:crying_cat_face: under what circumstances do we run installcheck against snapshot?

**Comment by Vogtinator** [comment]
When it runs against anything building against the snapshot, like I'm doing locally on `KDE:*` stuff and IIRC others on other projects as well.

**Comment by coolo** [comment]
Ah, RESOLVED/DONTDOTHAT :)

**Comment by Vogtinator** [comment]
What's the alternative?

**Comment by coolo** [comment]
building/testing against standard. Implementing this doesn't require magic, but considerable time investment.

**Comment by Vogtinator** [comment]
This was actually fairly simple to implement, `/usr/bin/installcheck` supports `primary.xml.gz` directly by itself.

I implemented this in https://github.com/Vogtinator/osc-plugin-factory/tree/dodmirror now, but noticed afterwards that the needed `--post-comments` option to repo-checker is gone... What's the alternative?

**Comment by coolo** [comment]
We meant to include the infos in build fail reminder - but never came around it.

**Comment by Vogtinator** [comment]
Fixed with #2403
